src/js/views/patients/worklist/filters_views.js

```diff
@@ -10,15 +10,26 @@ import 'js/views/patients/worklist/worklist-list.scss';
 const FiltersView = View.extend({
   className: 'worklist-list__filters',
   template: hbs`
+    <div class="worklist-list__all-filters" data-all-filters-region></div>
     <div class="worklist-list__filter" data-group-filter-region></div>
     <div class="worklist-list__filter" data-owner-toggle-region></div>
   `,
   regions: {
+    allFilters: '[data-all-filters-region]',
     group: '[data-group-filter-region]',
     ownerToggle: '[data-owner-toggle-region]',
   },
 });
 
+const AllFiltersButtonView = View.extend({
+  className: 'button--link worklist-list__all-filters-button',
```

Comment:
I missed the font-size change.  I think maybe we should make a `button--link-large` or something.. the icon styling isn't needed I think if the text is wrapped in a `span`